src/hooks/useDateDifferenceInDays.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const useDateDiffInDays = (date: string) => {
```

Comment:
this is a HOC. 

High Order Component !== Hooks

hooks encapsulate a business rule

and HOCs (High Order Components) are intelligent components

Reply:
change this to a atoms folder and remove 'use' from ur name.

Reply:
and u can use ''dayjs" here too, to make a more improve logic. 

example: https://day.js.org/docs/en/display/difference